src/bisect.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+async function bisect (compare, getByIndex, length) {
+  let top = length
+  let bottom = 0
+
+  async function _bisect () {
+    const middle = Math.floor((top + bottom) / 2)
+    const data = await getByIndex(middle)
+    const cmp = compare(data)
+
```

Comment:
why not handling also cmp === 0? https://github.com/bitfinexcom/hypercore-bisect/blob/master/index.js#L39

Reply:
yeah but we should keep _bisect generic and not tied to timestamp, I think in this case we should have a conditional branch also for equal